.github/ISSUE_TEMPLATE/bug_report.yml

```diff
@@ -1,6 +1,6 @@
 name: 🐞 Bug
 description: File a bug/issue
-title: "[BUG] <title>"
+title: '[BUG] <title>'
```

Comment:
I feel it's a bit overkill to have bug templates in this repo (but, it's not added in this PR so no worries)